W-22846824-rtf-india-cloud-hyperforce-kt (#1710)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-self-managed-network-configuration.adoc

To function correctly, Runtime Fabric requires the following hostname endpoint configurations. Give access to these endpoints in your infrastructure to avoid any installation or deployment issues.

=== US Cloud

This table lists the required hostname endpoint configurations for the US Cloud.

[%header,cols="4*a"]
|===
| Port | Protocol | Hostnames | Description

> | 443 | AMQP over WebSockets | `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the control plane.

Reviewer: ```suggestion
| 443 | AMQP over WebSockets | `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the cloud region.
```

| 443 (v1.8.50 and later) | TCP (Lumberjack) | `dias-ingestor-router.us-east-1.prod.cloudhub.io` +
`us1.ingest.mulesoft.com` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
| 5044 (legacy) | TCP (Lumberjack) | `dias-ingestor-nginx.prod.cloudhub.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *As of Runtime Fabric version 1.8.50, this port is not strictly required. It will be deprecated in the future.*

Reviewer: ```suggestion
*As of Runtime Fabric version 1.8.50, this port isn't strictly required. It will be deprecated in the future.*
```

| 443 | HTTPS | `anypoint.mulesoft.com` | Anypoint Platform for pulling assets.
| 443 | HTTPS | `worker-cloud-helm-prod.s3.amazonaws.com` | Runtime Fabric version repository. The Runtime Fabric installation uses software from this repository during installation and upgrades.
| 443 | HTTPS | `exchange2-asset-manager-kprod.s3.amazonaws.com` | Anypoint Exchange for application assets.
| 443 | HTTPS | `exchange-files.anypoint.mulesoft.com` | Anypoint Exchange for application files.
| 443 | HTTPS | `rtf-runtime-registry.kprod.msap.io` | Runtime Fabric Docker repository.
| 443 | HTTPS | `prod-us-east-1-starport-layer-bucket.s3.amazonaws.com`, `prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com` | Runtime Fabric Docker image delivery.
| 443 | HTTPS | `runtime-fabric.s3.amazonaws.com` | Runtime Fabric Docker repository.
| 443 | HTTPS | `configuration-resolver.prod.cloudhub.io` | Anypoint Configuration Resolver.
|===

=== EU Cloud

This table lists the required hostname endpoint configurations for the EU Cloud.

[%header,cols="4*a"]
|===
| Port | Protocol | Hostnames | Description

> | 443 | AMQP over WebSockets | `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.

Reviewer: ```suggestion
| 443 | AMQP over WebSockets | `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the cloud region.
```

| 443 (v1.8.50 and later) | TCP (Lumberjack) | `dias-ingestor-router.eu-central-1.prod-eu.msap.io` +
`eu1.ingest.mulesoft.com` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
| 5044 (legacy) | TCP (Lumberjack) | `dias-ingestor-nginx.prod-eu.msap.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.

> *As of Runtime Fabric version 1.8.50, this port is not strictly required. It will be deprecated in the future.*

Reviewer: ```suggestion
*As of Runtime Fabric version 1.8.50, this port isn't strictly required. It will be deprecated in the future.*
```

[%header,cols="4*a"]
|===
| Port | Protocol | Hostnames | Description

> | 11443 | WSS | `arm-mcm2-service-mulesoft-cp.sfdc-mchho0.svc.sfdcfc.net` | Runtime Fabric message broker for interaction with the control plane.

Reviewer: ```suggestion
| 11443 | WSS | `arm-mcm2-service-mulesoft-cp.sfdc-mchho0.svc.sfdcfc.net` | Runtime Fabric message broker for interaction with the cloud region.
```

| 8443 | HTTPS | `runtime-manager.jp1.platform.mulesoft.com` | Configuration Resolver, Exchange asset download, and Runtime Manager services. In the Japan Cloud, these services are consolidated behind a single endpoint.
| 8443 | HTTPS | `ingest.jp1.platform.mulesoft.com` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric (logs, metrics, and traces ingestion via OTEL).
| 443 | HTTPS | `rtf-runtime-registry-mulesoft-cp.sfdc-mchho0.svc.sfdcfc.net` | Runtime Fabric Docker and Helm chart repository. All container images (Mule runtime, monitoring sidecars, and Runtime Fabric agent) are pulled from this registry.
| 443 | HTTPS | `esvc-us-east-2-mulesoft.s3.us-east-2.amazonaws.com` | Runtime Fabric installer chart tarball. Required only for `rtfctl`-based installations.
| 443 | HTTPS | `prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com` | Runtime Fabric Docker image delivery.
|===

=== India Cloud

This table lists the required hostname endpoint configurations for the India Cloud.

[%header,cols="4*a"]
|===
| Port | Protocol | Hostnames | Description

> | 8443 | WSS | `runtime-manager.in1.platform.mulesoft.com` | Runtime Fabric message broker for interaction with the control plane (path: `/v1/connect`).

Reviewer: ```suggestion
| 8443 | WSS | `runtime-manager.in1.platform.mulesoft.com` | Runtime Fabric message broker for interaction with the cloud region (path: `/v1/connect`).
```
